ioSPI/osf_upload.py

```diff
@@ -0,0 +1,174 @@
+"""Module to upload data to OSF.io generated by simSPI TEM Simulator."""
+import typing
+from pathlib import Path
+
+import requests
+
+
+class TEMUpload:
+    """Class to upload data to OSF.io generated by simSPI TEM Simulator.
+
+    Parameters
+    ----------
+    token : str
+        Personal token from OSF.io with access to cryoEM dataset.
```

Comment:
could generalize `Personal token from OSF.io with access to dataset (e.g. cryoEM, etc)`.